src/lib.rs

```diff
@@ -147,27 +147,27 @@ trait RenderPassInner<Ctx: Context>: RenderInner<Ctx> {
 }
 
 trait Context: Sized {
-    type AdapterId: Send + Sync + 'static;
-    type DeviceId: Send + Sync + 'static;
-    type QueueId: Send + Sync + 'static;
-    type ShaderModuleId: Send + Sync + 'static;
-    type BindGroupLayoutId: Send + Sync + 'static;
-    type BindGroupId: Send + Sync + 'static;
-    type TextureViewId: Send + Sync + 'static;
-    type SamplerId: Send + Sync + 'static;
-    type BufferId: Send + Sync + 'static;
-    type TextureId: Send + Sync + 'static;
-    type PipelineLayoutId: Send + Sync + 'static;
-    type RenderPipelineId: Send + Sync + 'static;
-    type ComputePipelineId: Send + Sync + 'static;
-    type CommandEncoderId;
-    type ComputePassId: ComputePassInner<Self>;
-    type RenderPassId: RenderPassInner<Self>;
-    type CommandBufferId: Send + Sync;
+    type AdapterId: Debug + Send + Sync + 'static;
+    type DeviceId: Debug + Send + Sync + 'static;
+    type QueueId: Debug + Send + Sync + 'static;
+    type ShaderModuleId: Debug + Send + Sync + 'static;
+    type BindGroupLayoutId: Debug + Send + Sync + 'static;
+    type BindGroupId: Debug + Send + Sync + 'static;
+    type TextureViewId: Debug + Send + Sync + 'static;
+    type SamplerId: Debug + Send + Sync + 'static;
+    type BufferId: Debug + Send + Sync + 'static;
+    type TextureId: Debug + Send + Sync + 'static;
+    type PipelineLayoutId: Debug + Send + Sync + 'static;
+    type RenderPipelineId: Debug + Send + Sync + 'static;
+    type ComputePipelineId: Debug + Send + Sync + 'static;
+    type CommandEncoderId: Debug;
+    type ComputePassId: Debug + ComputePassInner<Self>;
+    type RenderPassId: Debug + RenderPassInner<Self>;
+    type CommandBufferId: Debug + Send + Sync;
     type RenderBundleEncoderId: RenderInner<Self>;
```

Comment:
Changes is `wgpu-core` are needed for `direct::Context::RenderBundleEncoderId` to implement `Debug`